EIPS/eip-969.md

```diff
@@ -0,0 +1,202 @@
+on the network, as well as the risk of providing too much advanced warning to 
+ASIC developers. 
+
+It is further understood that this change will not prevent redesign of existing
+dedicated hardware with new ASIC chips. The intention of this change is only
+to disable currently active or mid-production hardware and provide time for
+POS development as well as larger algorithim changes to be well analyzed by 
+experts.
+
+The choice of FNV constants is made based on the formal specification at
+https://tools.ietf.org/html/draft-eastlake-fnv-14#section-2.1
+
+@goobur provided the following python code to output primes matching this criteria
+
+`candidates = [2**24 + 2**8 + _ for _ in xrange(256)]`</br>  
+`candidates = [_ for _ in candidates if is_prime(_)]`</br>
+`["0x%x" % _ for _ in candidates if _ % (2**40 - 2**24 - 1) > (2**24 + 2**8 + 2**7)]`</br>  
+
+The minimal prime constraint was relaxed, as has already been the case in ethashV1.
+
+Typical ASIC synthesis tools would optimize multiplication of a constant
+in the FNV algorithm, while reducing the area needed for the multiplier according
+to the hamming weight of the constant. To reduce the chance of ASIC adaptation
+through minor mask changes, we propose choosing new constants with a larger
+hamming weight, however care should be taken not to choose constants with too
+large of a weight.
+
+The current FNV prime, 0x1000193 has a hamming weight of 6.   
+`HammingWeight(0x10001a7) = 7;`  
+`HammingWeight(0x10001ab) = 7;`    
+`HammingWeight(0x10001cf) = 8;`    
+`HammingWeight(0x10001e3) = 7;`    
+`HammingWeight(0x10001ef) = 9; // Not chosen`   
+`HammingWeight(0x10001f9) = 8;`  
+`HammingWeight(0x10001fb) = 9; // Not chosen`
+
+An exhaustive analysis was done regarding the dispersion of these constants as compared to 0x01000193 
+using the following process.
+
+```
+uint32_t candidate = 0;
+uint32_t dups = 0;
+uint32_t fnv_candidate = 0x10001a7;
+uint8_t *counts = malloc(0xFFFFFFFF);
+memset(counts, '\0', 0xFFFFFFFF);
+for ( candidate = 0; candidate <= 0xFFFFFFFF; candidate++) {
+    uint32_t result = (uint_32)(candidate * fnv_candidate);
+    uint8_t oldCount = counts[result];
+    counts[result] = counts[result]+1;
+    if (oldCount != 0) {
+       dups++;
+    }
+}
+printf("Candidate 0x%08x : %i dups", dups);
+```
+
+It can be empirically confirmed that no more than 1 duplicates occur in the 32 bit word space with these constants. 
+
+It is worth noting that FNV is not a cryptographic hash, and it is not used as such in ethash. With 
+that said, a more invasive hash algorithm change could consider other options. One suggestion has been 
+MurmorHash3 (https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp)
+
+Other suggestions have been made for complete algorithm changes, including off-the-shelf solutions 
+such as Argon2, Equihash of Zcash fame, and Balloon Hashing.
+(https://twitter.com/el33th4xor/status/981292363627810818). This may be considered once
```

Comment:
`[Other suggestions have been made](https://twitter.com/el33th4xor/status/981292363627810818)`.

Reply:
Yes, you need to create a new PR. It'll be automatically merged by the bot as soon as you submit it and the build passes.